util/cron/test-perf.chapcs.playground.bash

```diff
@@ -25,11 +25,11 @@ export CHPL_NIGHTLY_TEST_CONFIG_NAME="perf.chapcs.playground"
 # 4) Update START_DATE to be today, using the format mm/dd/yy
 #
 
-# Test not localizing non-primitives
-GITHUB_USER=bradcray
-GITHUB_BRANCH=no-localize-non-primitives
-SHORT_NAME=no-localize-non-primitives
-START_DATE=03/12/22
+# Test resize threshold of 0.75
+GITHUB_USER=bmcdonald3
+GITHUB_BRANCH=threshold-change
+SHORT_NAME=threshold-change
+START_DATE=04/19/22
```

Comment:
Not really important, but 04/29?

Reply:
Was hoping I could conjure up 10 extra days of data. Fixed